data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,753 @@
+# There be no copyright. These treasures be dedicated to the public domain by Galaucus, me hearties.
+#
+# CC0 Public Domain Dedication
+#
+# The person who associated a work with this deed has dedicated the work to the public domain
+# by waiving all of his or her rights to the work worldwide under copyright law, including all
+# related and neighboring rights, to the extent allowed by law.
+#
+# You can copy, modify, distribute and perform the work, even for commercial purposes, all
+# without asking permission.
+#
+# In no way are the patent or trademark rights of any person affected by CC0, nor are the
+# rights that other persons may have in the work or in how the work is used, such as publicity
+# or privacy rights.
+#
+# Unless expressly stated otherwise, the person who associated a work with this deed makes no
+# warranties about the work, and disclaims liability for all uses of the work, to the fullest
+# extent permitted by applicable law.
+#
+# When using or citing the work, you should not imply endorsement by the author or the affirmer.
+
+
+mission "You Are A Pirate"
+	landing
+	invisible
+	source
+		government "Pirate"
+	to offer
+		not "chosen sides"
+		or
+			and
+				or
+					"reputation: Republic" < -10
+					"reputation: Syndicate" < -10
+					"reputation: Free Worlds" < -10
+				"reputation: Merchant" < -10
+			"reputation: Pirate" > 20
+	on offer
+		conversation "You Are A Pirate"
+	on accept
+		event "became a pirate"
+		fail
```

Comment:
We don't need to trigger this event in the on accept that immediately fails the mission anymore. We can instead use a conversation action to trigger the event after you've made the conversation choice that would lead to accepting the mission currently. We'd then always decline the mission.